stdlib/public/stubs/GlobalObjects.cpp

```diff
@@ -41,6 +49,62 @@ swift::_SwiftEmptyArrayStorage swift::_swiftEmptyArrayStorage = {
   }
 };
 
+
+
+swift::_SwiftEmptyDictionaryStorage swift::_swiftEmptyDictionaryStorage = {
```

Comment:
Also: `(void*)1` is illegal in a constexpr. I gave up and just made it nullptr with the understanding that there's no conceivable way a null non-nullable pointer that is never looked at could matter.